### DISCUSSION TITLE
q-table selection checkboxes on right side

### DISCUSSION THREAD

**Discussion by juniorcoder618** [quasarframework/quasar#9128] [General - Components / Directives / etc]
how can we make the default (left) selection checkboxes to be at the end of the table (right side)

![image](https://user-images.githubusercontent.com/83473765/116744597-88497180-a9c8-11eb-841d-aeecd6b727c6.png)

**Comment by ontwikkelfabriek** [comment]
Here's the only way I can think of with using the qtable's own selected( I cheated):
https://codepen.io/ontwikkelfabriek/pen/BapgOav

What you could actually do is use the `body-cell-actions` slot to put in a custom checkbox. But then you'll not
be using the qtable's selection,  and you'll have to do some custom coding.

Here's a start but you'll have to create your own selection logic:
https://codepen.io/ontwikkelfabriek/pen/WNRqgje
